docs/overview.md

```diff
@@ -21,19 +21,19 @@ When the tx bytes are sent to a validator, the validator executes the function V
 
-Our ante handler will check that the address that created the signatures in BaseTx match those that own the inputs of the transaction as well as check that the inputs being spent exist. The ante handler also checks that the inputs = outputs + fee. 
+Our ante handler will check that the address that created the signatures in BaseTx match those that own the inputs of the transaction as well as check that the inputs being spent exist. The ante handler also checks that the inputs = outputs + fee. TxConfirmationSigantures will also be checked for each input that is not a deposit nor a fee utxo.  
 
 Once a transaction has been included into a block, DeliverTx will be executed which will do the same functionality as CheckTx as well as route the Msg to a handler. 
 
-In our implementation, our handler can only handle a SpendMsg. Handling of a SpendMsg includes deleting the inputs of the transaction from our utxo database and creating new utxos corresponding to the addresses and denominations provided in SpendMsg. handleSpendMsg() will utilize the UTXOKeeper which is used to restrict access to our UTXOMapper (utxo database). The handler will also increment the transaction index number so the newly created utxo's will map from the correct position.
+Our implementation utilizes our utxo module in x/utxo. The utxo module allows for modularity when creating a utxo based system. We use our BaseUTXO to implement the UTXO interface. In addition to the base traits of a UTXO, we also keep the MsgHash, TxHash, and InputAddresses in our UTXO. The ProtoUTXO() function allows for our extra information to be stored when the handler in x/utxo creates a new output utxo.
 
 The UTXOMapper is our utxo database. Our mapper uses keys in the form: < encoded address > + < encoded position > . It maps to the encoded utxo and uses go-amino for its encoding. The < encoded position > at the beginning of the key is used for prefix iteration which will return all the utxo's owned by a specified address. 
 
 ## Types
 
 **SpendMsg**
 
-A SpendMsg contains the position of the input utxos (block number, transaction index, output index, deposit number), confirm signatures signed by the owners of the parent inputs (the inputs to the SpendMsg inputs), the addresses of the input owners, the addresses of the outputs, the denomination of each output, and the fee amount. 
+A SpendMsg contains the position of the input utxos (block number, transaction index, output index, deposit number), transaction confirmation signatures signed by the owners of the parent inputs (the inputs to the SpendMsg inputs), the addresses of the outputs, the amount of each output, and the fee amount. 
```

Comment:
input confirmation signatures not transaction confirmation signatures

Reply:
fixed